api/v1/certificatepolicy_types.go

```diff
@@ -69,73 +66,121 @@ func (t Target) String() string {
 	return fmt.Sprintf(fmtSelectorStr, t.Include, t.Exclude, *t.MatchLabels, *t.MatchExpressions)
 }
 
-// CertificatePolicySpec defines the desired state of CertificatePolicy
+// CertificatePolicySpec defines the certificates to be checked and the parameters within which
+// those certificates should fall.
 type CertificatePolicySpec struct {
-	// Only Inform is currently supported. Setting this to Enforce will have the same effect as Inform.
-	// +kubebuilder:validation:Enum=Inform;inform;Enforce;enforce
+	// Set to "Inform". Enforce is not currently supported, so setting this to Enforce will have the
+	// same effect as Inform.
 	RemediationAction RemediationAction `json:"remediationAction,omitempty"`
-	// 'namespaceSelector' defines the list of namespaces to include/exclude for objects defined in
-	// spec.objectTemplates. All selector rules are ANDed. If 'include' is not provided but
-	// 'matchLabels' and/or 'matchExpressions' are, 'include' will behave as if ['*'] were given. If
-	// 'matchExpressions' and 'matchLabels' are both not provided, 'include' must be provided to
+
+	// NamespaceSelector defines the list of namespaces to include/exclude for objects defined in
+	// spec.objectTemplates. All selector rules are ANDed. If Include is not provided but
+	// MatchLabels and/or MatchExpressions are, Include will behave as if ['*'] were given. If
+	// MatchExpressions and MatchLabels are both not provided, Include must be provided to
 	// retrieve namespaces.
 	NamespaceSelector Target `json:"namespaceSelector,omitempty"`
-	// Restrict the secrets that will be checked to the ones that contain these labels.
+
+	// LabelSelector restricts the secrets that will be checked to the ones that match these labels.
 	LabelSelector map[string]NonEmptyString `json:"labelSelector,omitempty"`
-	// low, medium, high, or critical
+
+	// Severity is a user-defined severity for when a certificate is found out of compliance with this
+	// certificate policy. Accepted values are low, medium, high, and critical.
+	//
 	// +kubebuilder:validation:Enum=low;Low;medium;Medium;high;High;critical;Critical
 	Severity string `json:"severity,omitempty"`
-	// Minimum duration before a certificate expires that it is considered non-compliant. Golang's time units only.
+
+	// MinDuration is the minimum duration for a certificate's expiration where any shorter duration
+	// is considered non-compliant. The value follows the Golang duration format 0h0m0s where hours
+	// (h) is the greatest accepted unit of time.
```

Comment:
Updated!

Right, the units are a Golang thing:
```
// There is no definition for units of Day or larger
// to avoid confusion across daylight savings time zone transitions.
```
https://cs.opensource.google/go/go/+/refs/tags/go1.22.1:src/time/time.go;l=625